src/package/package.js

```diff
@@ -707,8 +719,17 @@ module.exports = {
     }
 
     if (filesCount === existCount) {
-      apmJson.removePackage(instPath, uninstalledPackage);
-      await this.setPackagesList(instPath, true);
+      if (!uninstalledPackage.id.startsWith('script_')) {
+        apmJson.removePackage(instPath, uninstalledPackage);
+        await this.setPackagesList(instPath, true);
+      } else {
+        apmJson.removePackage(instPath, uninstalledPackage);
```

Comment:
There are the same processes.

```suggestion
      apmJson.removePackage(instPath, uninstalledPackage);
      if (!uninstalledPackage.id.startsWith('script_')) {
        await this.setPackagesList(instPath, true);
      } else {
```